examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/OfferingFragment.kt

```diff
@@ -57,14 +58,13 @@ class OfferingFragment : Fragment(), PackageCardAdapter.PackageCardAdapterListen
         return binding.root
     }
 
-    override fun onPurchasePackageClicked(cardView: View, currentPackage: Package) {
+    override fun onPurchasePackageClicked(cardView: View, currentPackage: Package, purchaseOption: PurchaseOption?) {
         binding.purchaseProgress.visibility = View.VISIBLE
-        val basePlanOption = currentPackage.product.purchaseOptions.first { it.isBasePlan }
 
         Purchases.sharedInstance.purchasePackageOptionWith(
             requireActivity(),
             currentPackage,
-            basePlanOption,
+            purchaseOption!!, // TODOBC5: Fix API for OTP
```

Comment:
So we still have to change the API to support OTP if they are not going to have any `PurchaseOption`. Will do that in a followup PR.